pkg/node/local_node.go

```diff
@@ -15,14 +15,17 @@
 package node
 
 import (
-	"github.com/cilium/cilium/pkg/option"
-
 	"time"
 
+	"github.com/cilium/cilium/pkg/option"
+
 	"k8s.io/api/core/v1"
 )
 
-var localNode Node
+var (
+	localNode      Node
+	nodeRegistered = make(chan struct{})
```

Comment:
This is now a global variable, so if `ConfigureLocalNode()` is called more than once, e.g. in unit tests, `close(nodeRegistered)` panics.